<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuBar</class>
 <widget class="QWidget" name="MenuBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>61</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #00838F</string>
  </property>
  <widget class="QPushButton" name="homeButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

	border: none;
	background-color: #26C6DA;
}

QPushButton:pressed{

	background-color: #00ACC1;
}

QPushButton:checked{

	background-color: #00ACC1;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/Icons/UI/Icons/sharp-home-24px.svg</normaloff>:/Icons/UI/Icons/sharp-home-24px.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="verticalLine">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>1</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #000000</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="startAPrintButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>122</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

	border: none;
	background-color: #26C6DA;
}

QPushButton:pressed{

	background-color: #00ACC1;
}

QPushButton:checked{

	background-color: #00ACC1;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/Icons/UI/Icons/sharp-play_arrow-24px.svg</normaloff>:/Icons/UI/Icons/sharp-play_arrow-24px.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="horizontalLine1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>60</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #000000</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="infoButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>61</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

	border: none;
	background-color: #26C6DA;
}

QPushButton:pressed{

	background-color: #00ACC1;
}

QPushButton:checked{

	background-color: #00ACC1;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/Icons/UI/Icons/sharp-info-24px.svg</normaloff>:/Icons/UI/Icons/sharp-info-24px.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="logButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>183</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

	border: none;
	background-color: #26C6DA;
}

QPushButton:pressed{

	background-color: #00ACC1;
}

QPushButton:checked{

	background-color: #00ACC1;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/Icons/UI/Icons/sharp-format_align_justify-24px.svg</normaloff>:/Icons/UI/Icons/sharp-format_align_justify-24px.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="settingsButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>244</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

	border: none;
	background-color: #26C6DA;
}

QPushButton:pressed{

	background-color: #00ACC1;
}

QPushButton:checked{

	background-color: #00ACC1;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/Icons/UI/Icons/sharp-settings-24px.svg</normaloff>:/Icons/UI/Icons/sharp-settings-24px.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="sysActionsButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>540</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

	border: none;
	background-color: #26C6DA;
}

QPushButton:pressed{

	background-color: #00ACC1;
}

QPushButton:checked{

	background-color: #00ACC1;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/Icons/UI/Icons/sharp-power_settings_new-24px.svg</normaloff>:/Icons/UI/Icons/sharp-power_settings_new-24px.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="Line" name="horizontalLine2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>121</y>
     <width>60</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #000000</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="horizontalLine3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>182</y>
     <width>60</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #000000</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="horizontalLine4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>243</y>
     <width>60</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #000000</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="horizontalLine5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>304</y>
     <width>60</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #000000</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="horizontalLine6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>539</y>
     <width>60</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #000000</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
